docs/getting_started_with_jax_for_AI.ipynb

```diff
@@ -6,7 +6,7 @@
     "id": "AEQPh3NtawWA"
    },
    "source": [
-    "# Getting started with JAX\n",
+    "# Getting started with JAX for AI\n",
```

Comment:
As discussed, added emphasis on "for AI" - we have the JAX quickstart (separate docs).